devolearn/tests/test.py

```diff
@@ -4,6 +4,7 @@
 import pandas as pd 
 import matplotlib.pyplot as plt
 import os
+import pytest
```

Comment:
is importing `pytest` and using the decorators below really required ? I've seen implementations where `pytest `was not even imported and the tests worked fine with `pytest filename.py`

Reply:
1. The markers/decorators will enable us to run a subset of the test suite. 
2. To use these markers, we have to import the pytest module in the test file.

For example, if a user wants to run tests only for the lineage population model, he'll use `pytest -m lineage_population_model -v`